inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/es/ElasticsearchConfig.java

```diff
@@ -104,4 +104,45 @@ private void setEsAuth(RestClientBuilder builder) {
             logger.error("set es auth error ", e);
         }
     }
+
+    public void setHost(String arg) {
```

Comment:
Just add the `@Data` annotation in the class, no need to add every set and get methods.

Reply:
OK, changed